report/result.go

```diff
@@ -128,11 +128,11 @@ func ToHTML(r probe.Result) string {
 					<td class="data">%s</td>
 				</tr>
 			</table>
-		` + HTMLFooter(r.StartTime.Format(r.TimeFormat))
+		` + HTMLFooter(FormatTime(r.StartTime))
```

Comment:
A quick question: Who should take the responsibility for rendering the the time on the web page?  the server or the client? It has a little different from other notifiers.

Reply:
currently, the web server renders the whole web page including the time.